ghtt/search.py

```diff
@@ -17,7 +17,7 @@ def notify(api_key, domain_name, to, repos, query):
     text = ""
     for g_repo in repos:
         text = text + g_repo.html_url
-        g_commit = g_repo.get_branch("master").commit
+        g_commit = g_repo.get_branch(g_repo.master_branch or "master").commit
```

Comment:
Use `default_branch` instead of `master_branch`. Every repo should have a `default_branch` so no fallback needed.

Note: we don't use our own `default_branch`, since the `search` subcommand doesn't use `ghtt.yaml` and returns random repos from other users/orgs.

```suggestion
        g_commit = g_repo.get_branch(g_repo.default_branch).commit
```